src/app/modules/contract/contract-form/contract-form.component.scss

```diff
@@ -96,3 +96,14 @@ mat-card-content {
         flex: 2 0 0;
     }
 }
+
+.price-fields {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    gap: 1rem;
```

Comment:
You could reuse mat-card-content instead in line 19

Reply:
mat-card-content {  => mat-card-content, .price-fields {

Reply:
Gotcha!